contents/handbook/company/team/paolo-damico.md

```diff
@@ -1,22 +1,35 @@
 ---
-title: Paolo D'Amico's README
+title: Paolo D'Amico's Goals
 sidebar: Handbook
 showTitle: true
 ---
 
-## Goals (6 weeks until 6 Aug 2021)
+## Q4 2021 Goals
 
-*   **Execute on nailing funnels.** Ensure the funnels product actually solves user’s needs and is shipped with the highest user experience quality.
-    * Ensure engineers have all the context needed to Nail Funnels, particularly information around user’s problems / use cases, user feedback, and relevant data insights.
-    * Support engineers in scoping problems so they maximize for value delivered and can be built in a constrained time frame.
-    * Provide feedback, test features thoroughly and report/fix issues based on dogfooding, user feedback and usability tests to ensure a high quality experience.
-    * Support Marcus and the entire team measure success through qualitative, quantitative metric(s).
-*   **On what’s next.** Provide as much context as possible (mainly on user input, usage data, and user experience intuition) for engineers and the Exec Team to decide what to work on next (on an operational and strategic level, respectively).
-    * Support engineers set up their own behavioral metrics and encourage them to measure results on their own work and figure out improvements.
-    * Provide context from user interviews, data analyses and other sources to figure out sprint priorities. Includes helping break down problems.
-    * Provide internal product context to non-engineering members of the team as we figure out what to focus on next. 
-*   **Engineering**
-    *   Ship series/steps naming feature OR saved reports.
-    *   Ship UI/UX improvements on funnels as features are built.
-    *   Pick up any critical bug or UX issue if no one else can.
+- **Product strategy.**
+  - ⭐️ Build the _core product_ strategy for Q4-21 (particularly related to Collaboration & Experimentation).
+  - Support Exec Team with context, feedback and any other activity in shaping up the product strategy for the next quarter/year. In particular, ensure we have clear company-level metrics and we have a plan for improving them.
```

Comment:
I would say non-product company metrics e.g. revenue should be outside of the scope of this goal

```suggestion
  - Support Exec Team with context, feedback and any other activity in shaping up the product strategy for the next quarter/year. In particular, ensure we have clear company-level product metrics and we have a plan for improving them.
```